src/main/scala/com/microsoft/hyperspace/index/CachingIndexCollectionManager.scala

```diff
@@ -40,11 +42,11 @@ class CachingIndexCollectionManager(
     indexLogManagerFactory: IndexLogManagerFactory,
     indexDataManagerFactory: IndexDataManagerFactory,
     fileSystemFactory: FileSystemFactory)
-    extends IndexCollectionManager(
-      spark,
-      indexLogManagerFactory,
-      indexDataManagerFactory,
-      fileSystemFactory) {
+  extends IndexCollectionManager(
+    spark,
+    indexLogManagerFactory,
+    indexDataManagerFactory,
+    fileSystemFactory) {
```

Comment:
Can you use the scalafmt to format: https://github.com/microsoft/hyperspace/blob/master/docs/coding-guidelines/scala-coding-style.md? Applies to other files.